front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -210,34 +210,17 @@ const Moderation = memo<Props & InjectedIntlProps>(({ className, intl }) => {
     },
   ];
 
-  const {
-    list,
-    pageSize,
-    moderationStatus,
-    currentPage,
-    lastPage,
-    onModerationStatusChange,
-    onPageNumberChange,
-    onPageSizeChange,
-    onModeratableTypesChange,
-    onProjectIdsChange,
-    onSearchTermChange,
-    onIsFlaggedChange,
-  } = useModerations({
-    pageSize: pageSizes[1].value,
-    moderationStatus: 'unread',
-    moderatableTypes: [],
-    projectIds: [],
-    searchTerm: '',
-  });
-
-  const [moderations, setModerations] = useState(list);
   const [selectedModerations, setSelectedModerations] = useState<
     IModerationData[]
   >([]);
   const [processing, setProcessing] = useState(false);
   const [selectedTypes, setSelectedTypes] = useState<TModeratableTypes[]>([]);
   const [selectedProjectIds, setSelectedProjectIds] = useState<string[]>([]);
+  const [selectedPageNumber, setSelectedPageNumber] = useState<number>(1);
```

Comment:
Added all these to the state because I changed the way we deal with passing the params to useModerations (see below).

Reply:
Aren't all of these best kept in the url? Not saying we should change it now but wondering why we opted to keep them in component state in the first place.

Reply:
Good question. We don't work with visible parameters in the URL very often. CDM material, perhaps? :-)